waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -1667,7 +1646,7 @@ public GetTableResult get_table_req(GetTableRequest req) throws MetaException, N
   @Override
   @Loggable(value = Loggable.DEBUG, skipResult = true, name = INVOCATION_LOG_NAME)
   public GetTablesResult get_table_objects_by_name_req(GetTablesRequest req)
-      throws MetaException, InvalidOperationException, UnknownDBException, TException {
+      throws TException {
```

Comment:
I'm sorry but I think we should revert the changes to this entire class so it's more in line with the class its implementing.

Reply:
Ok